docker.Makefile

```diff
@@ -71,11 +77,11 @@ e2e-cross: create_bin
 	@$(call chmod,+x,bin/$(BIN_NAME)-e2e-windows.exe)
 
 tars:
-	tar czf bin/$(BIN_NAME)-linux.tar.gz -C bin $(BIN_NAME)-linux
+	tar --transform='flags=r;s|$(BIN_NAME)-linux|$(BIN_NAME)-plugin-linux|' -czf bin/$(BIN_NAME)-linux.tar.gz -C bin $(BIN_NAME)-linux ${BIN_STANDALONE_NAME}-linux
```

Comment:
This probably warrants an update to the README.md cf the install instructions which should mention the need to rename the `docker-app-plugin-x` to just `docker-app` and where to put it.

Reply:
Yep right need to make a pass on the README 👍

Reply:
Done